src/main/java/com/mongodb/jdbc/MongoDatabaseMetaData.java

```diff
@@ -223,7 +223,7 @@ public class MongoDatabaseMetaData implements DatabaseMetaData {
     private static final List<SortableBsonDocument.SortSpec> GET_INDEX_INFO_SORT_SPECS =
             Arrays.asList(
                     new SortableBsonDocument.SortSpec(
-                            NON_UNIQUE, SortableBsonDocument.ValueType.String),
+                            NON_UNIQUE, SortableBsonDocument.ValueType.Boolean),
                     new SortableBsonDocument.SortSpec(
                             INDEX_NAME, SortableBsonDocument.ValueType.String),
                     new SortableBsonDocument.SortSpec(
```

Comment:
Once I fixed the first issue, I got the following error: `Error reading table indexes: SQL Error: org.bson.BsonInvalidOperationException: Value expected to be of type STRING is of unexpected type BOOLEAN`. This was due to `NON_UNIQUE` holding a Boolean rather than a String like we said it would in the sort specs, so I updated the sort specs and `SortableBsonDocument.java` to handle booleans.